Core/src/RequestWrapper.php

```diff
@@ -373,13 +373,6 @@ private function getExceptionMessage(\Exception $ex)
     {
         if ($ex instanceof RequestException && $ex->hasResponse()) {
             return (string) $ex->getResponse()->getBody();
-
-            try {
```

Comment:
What's the driver of this change?

Reply:
It's dead code. The immediately preceding line is a return.